### PR TITLE
VizRepeater: Prevent opposite scrollbar

### PR DESCRIPTION
**Pull request by drew08t**
For VizRepeater component, in some browsers, the presence of a scrollbar in the opposite direction of panel orientation causes content to overflow, creating another scrollbar. This PR prevents overflow in the same direction as the orientation direction.

Before:
image

After:
image

For VizOrientation.Horizontal, prevent horizontal scrollbar.
For VizOrientation.Vertical, prevent vertical scrollbar.

Fixes https://github.com/grafana/support-escalations/issues/12644

### CODE REVIEW

**Comment by drew08t** [comment]
We may want to add some more padding inside of the viz repeater content, as some content might get cutoff with scroll.

File: packages/grafana-ui/src/components/VizRepeater/VizRepeater.tsx [hunk 1]
```diff
@@ -180,13 +180,15 @@ export class VizRepeater<V, D = {}> extends PureComponent<PropsWithDefaults<V, D
         const defaultVizHeight = (height + itemSpacing) / values.length - itemSpacing;
         repeaterStyle.flexDirection = 'column';
         repeaterStyle.height = `${height}px`;
+        repeaterStyle.overflowX = `hidden`;
```
Comment (adela-almasan) [inline packages/grafana-ui/src/components/VizRepeater/VizRepeater.tsx:183]:
```suggestion
        repeaterStyle.overflowX = 'hidden';
```

File: packages/grafana-ui/src/components/VizRepeater/VizRepeater.tsx [hunk 2]
```diff
@@ -180,13 +180,15 @@ export class VizRepeater<V, D = {}> extends PureComponent<PropsWithDefaults<V, D
...
         itemStyles.marginBottom = `${itemSpacing}px`;
         vizWidth = width;
         vizHeight = clamp(defaultVizHeight, minVizHeight ?? 0, maxVizHeight ?? defaultVizHeight);
         break;
       case VizOrientation.Vertical:
         repeaterStyle.flexDirection = 'row';
         repeaterStyle.justifyContent = 'space-between';
+        repeaterStyle.overflowY = `hidden`;
```
Comment (adela-almasan) [inline packages/grafana-ui/src/components/VizRepeater/VizRepeater.tsx:191]:
```suggestion
        repeaterStyle.overflowY = 'hidden';
```

File: packages/grafana-ui/src/components/VizRepeater/VizRepeater.tsx (hunk 1 above)
Comment (drew08t) [reply packages/grafana-ui/src/components/VizRepeater/VizRepeater.tsx:183]: Good catch!

**Review by adela-almasan — APPROVED** [review]
Code looks good! I can't repro the issue locally so I can't test it though.

**Review by Develer — APPROVED** [review]

**Pull request merged**